source/drivers/cxx.txt

```diff
@@ -0,0 +1,87 @@
+.. include:: /includes/unicode-checkmark.rst
+
+==================
+MongoDB C++ Driver
+==================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+Introduction
+------------
+
+The MongoDB C++ Driver is the official client library for >C++11 applications.
```

Comment:
Did you mean "greater than" C++11? If so I'd reword it to `The MongoDB C++ Driver is the official client library for C++11 applications and later.